src/main/java/no/nav/pto/veilarbportefolje/persononinfo/Landgruppe.java

```diff
@@ -0,0 +1,304 @@
+package no.nav.pto.veilarbportefolje.persononinfo;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Landgruppe {
+    private static Map<String, String> landGruppe0 = Stream.of(new String[][]{
+            {"NOR", "Norge"},
+            {"SJM", "Svalbard og Jan Mayen"}
+    }).collect(Collectors.toMap(data -> data[0], data -> data[1]));
+
+    private static Map<String, String> landGruppe1 = Stream.of(new String[][]{
```

Comment:
Could this be a .json file?

Reply:
And maybe it would be better to get the name of a country from kodeverket?

Reply:
Yep yep, that will be next improvement.